src/addons/moziot-adapter-gpio/gpio-adapter.js

```diff
@@ -105,18 +105,18 @@ class GpioDevice extends Device {
 }
 
 class GpioAdapter extends Adapter {
```

Comment:
There are some references to adapterManager in addons-test/moziot-adapter-settings/settings-adapter.js that should be changed to addonManager.